packages/server/src/utilities/rowProcessor/attachments.ts

```diff
@@ -25,6 +19,27 @@ export class AttachmentCleanup {
     }
   }
 
+  private static extractAttachmentKeys(
+    type: FieldType,
+    rowData: any
```

Comment:
Would it make sense to type this `rowData: { key: string }`?

Reply:
It might also be an array, and the rows are not typed at all. Also, the shape would need to depend on the FieldType, making the types more messy for a private function. So I think it's not really worth it in this case